build.fsx

```diff
@@ -159,7 +159,7 @@ Target "PrepAppConfig" <| fun _ ->
     let ip = environVarOrNone "container_ip"
     match ip with
     | Some ip ->
-        let appConfig = "src/Akka.Persistence.SqlServer.Tests/App.config"
+        let appConfig = "src/Akka.Persistence.SqlServer.Tests/bin/Release/Akka.Persistence.SqlServer.Tests.dll.config"
```

Comment:
Alter the `Akka.Persistence.SqlServer.Tests.dll.config` file only instead of the app.config during Docker-based test runs so contributors don't have to commit the app.config changes constantly